keyboards/jcpm2/config.h

```diff
@@ -0,0 +1,27 @@
+/* Copyright 2025 Jeremy Cook Consulting LLC <info@jeremyscook.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * If you did not receive a copy of the GNU General Public License
+ * along with this program, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+// I2C Configuration
+#define I2C_DRIVER I2CD1
+#define I2C1_SCL_PIN D0
+#define I2C1_SDA_PIN D1
+
+// OLED Configuration
+#define OLED_DISPLAY_128X64
+#define OLED_DISPLAY_ADDRESS 0x3C
+#define OLED_TIMEOUT 2000000
```

Comment:
Default behaviour(s)
```suggestion
#define OLED_DISPLAY_128X64
#define OLED_TIMEOUT 2000000
```